docs/docs/configuration/providers/azure_entra_oidc.md

```diff
@@ -0,0 +1,187 @@
+---
+id: azure_entra_oidc
+title: Azure Entra ID OIDC
```

Comment:
Wouldn't the correct name be Microsoft Entry ID?
https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-id

Reply:
Yes it is. Changed now in documentation

Reply:
Any preferences how to name the provider setting (currently `azure-oidc`). `ms-entra-id` could be an option.

Reply:
I believe it's technically [Microsoft identity platform](https://learn.microsoft.com/en-us/entra/identity-platform/v2-overview) as it can authenticate more than just Entra users.

![about-microsoft-identity-platform](https://github.com/oauth2-proxy/oauth2-proxy/assets/16367391/1ac67fed-d6d1-47c8-b013-d1cba191ce7c)

Reply:
I think MS Entra ID name fits more into here, this is purely a successor of Azure AD, you can configure it fully from Azure through App registration as it was before.